muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/organizationmanagement/rest/OrganizationUserManagementRESTService.java

```diff
@@ -307,7 +307,8 @@ public Response searchStudents(
             userEntity.getUpdatedByStudent(),
             userEntity.getId(),
             null, // flags
-            organizationRESTModel));
+            organizationRESTModel,
+            null));
```

Comment:
I'd prefer `Boolean.FALSE` over `null`, when dealing with boolean values. Same comment applies to multiple classes in this PR.